integration-test-groups/aws2/aws2-s3/src/test/java/org/apache/camel/quarkus/component/aws2/s3/it/Aws2S3Test.java

```diff
@@ -41,6 +44,9 @@
     private int objects_num_before;
     private int objects_num_after;
 
+    @Aws2Client(LocalStackContainer.Service.S3)
+    S3Client client;
```

Comment:
We inject a client without using it directly. Is there a trick to explain ?

Reply:
It is requirement for client created via quarkus, that the client has to be injected (doesn't need to be used). The injection point is detected by quarkus and client is created. If there is no injection point, client is not created even if all required paameters are present (https://camel.apache.org/camel-quarkus/2.6.x/reference/extensions/aws2-s3.html#_optional_integration_with_quarkus_amazon_s3)

Reply:
sorry I missed it from my older attempts, I'll remove this one variable

Reply:
Ok, I think you are right that it's slated for removal.

On the other side, reading again the doc section pointed in previous comment, I have another question somewhere else in the PR :)